recipe/meta.yaml

```diff
@@ -49,4 +48,6 @@ about:
 
 extra:
     recipe-maintainers:
+        - danclewley
+        - gillins
```

Comment:
@danclewley and @gillins I hope that is OK. If not I can remove you from the list.